### PR TITLE
DIO-109 go cli interactive functions

### PR DESCRIPTION
**Pull request by MattyJ007**

### CODE REVIEW

File: goinstant/prompts.go [hunk 1]
```diff
@@ -81,57 +180,382 @@ func selectPackageDocker() {
...
+		fmt.Println("Stopping and Cleaning Up Electronic Medical Record...")
+		RunDirectDockerCommand([]string{"docker", "emr", "destroy"})
+		selectDefaultInstall()
+
+	case "Stop and Cleanup Health Management Information System":
+		fmt.Println("Stopping and Cleaning Up Health Management Information System...")
+		RunDirectDockerCommand([]string{"docker", "hmis", "destroy"})
+		selectDefaultInstall()
+
+	case "Stop and Cleanup Health Worker":
```
Comment (michaelloosen) [inline goinstant/prompts.go:539]: Is this not duplicating the logic under "Stop and Cleanup Workforce"?

**Review by michaelloosen — CHANGES_REQUESTED** [review]

File: goinstant/prompts.go [hunk 2]
```diff
@@ -81,57 +180,382 @@ func selectPackageDocker() {
...
+		selectCustomOptions()
+	}
+}
+
+func executeCommand() {
+	startupCommands := []string{"docker", customOptions.startupAction}
+	if customOptions.startupPackages != nil && len(customOptions.startupPackages) > 0 {
+		startupCommands = append(startupCommands, customOptions.startupPackages...)
+	} else {
+		fmt.Println("No startup package specified, cannot start.")
```
[the remark this answers is not in the document]
Comment (MattyJ007) [reply goinstant/prompts.go:198]: Perhaps remove it for consistency. I don't think anyone will use the init all packages option in reality - but the destroy all is very useful (especially in testing) sometimes I start up components in odd projects then just want to clean up everything without much typing 🙃

File: goinstant/prompts.go (hunk 1 above)
Comment (MattyJ007) [reply goinstant/prompts.go:539]: Health Workforce and Health Worker are different components

Comment (michaelloosen) [reply goinstant/prompts.go:539]: Indeed, but both have exactly the same docker command, so what am I missing here? 😁

Comment (michaelloosen) [reply goinstant/prompts.go:539]: Indeed, so they should have different docker commands right? 😁

Comment (MattyJ007) [reply goinstant/prompts.go:542]: lol, I didn't expand the whole file :p thought you were confused

Comment (michaelloosen) [reply goinstant/prompts.go:542]: Cool, thanks for correcting it 🙂

**Pull request merged**